docs/src/main/tut/client.md

```diff
@@ -87,7 +90,7 @@ the world" varies by context:
 * Here in the REPL, the last line is the end of the world.  Here we go:
 
 ```tut:book
-greetingList.unsafePerformSync.mkString("\n")
+//greetingList.unsafeRun.mkString("\n")
 ```
```

Comment:
If I actually call unsafeRun on this, the tut never finishes compiling, so I figure the Task never completes. I haven't tracked this down, yet.